django/api/services/spreadsheet_uploader_prep.py

```diff
@@ -302,6 +304,8 @@ def email_validator(df, *columns, **kwargs):
             try:
                 validate_email(value, dns_resolver=resolver)
             except EmailNotValidError:
+                if pd.notna(value) and value != '':
```

Comment:
Same as above; if we want empty values to not register as errors, we have to use the negation instead:
```
for index, value in series.items():
            if pd.isna(value) or value == '':
                continue # skip the rest of the loop and go to the next iteration
            try:
                validate_email(value, dns_resolver=resolver)
            except EmailNotValidError:
                indices.append(index + kwargs.get("indices_offset", 0))
```